tests/cross_functional/conftest.py

```diff
@@ -1591,3 +1591,64 @@ def factory(
         logger.info(f"No {search_string} errors are found in the noobaa pod logs")
 
     return factory
+
+
+@pytest.fixture(scope="session")
+def setup_stress_testing_bucket(bucket_factory_session, rgw_bucket_factory_session):
```

Comment:
We're returning multiple buckets, so:
```suggestion
def setup_stress_testing_buckets(bucket_factory_session, rgw_bucket_factory_session):
```